app/Resources/views/admin/blog/index.html.twig

```diff
@@ -20,8 +20,8 @@
                 {# it's not mandatory to set the timezone in localizeddate(). This is done to
                    avoid errors when the 'intl' PHP extension is not available and the application
                    is forced to use the limited "intl polyfill", which only supports UTC and GMT #}
-                <td>{% if post.publishedAt %}{{ post.publishedAt|localizeddate('short', 'short', null, 'UTC') }}{% endif %}</td>
-                <td>
+                <td>{{ post.publishedAt|localizeddate('medium', 'short', null, 'UTC') }}</td>
```

Comment:
Apparently that case is not possible. If a post exists, it's published (the date can be in the future) and this field is not nullable in the database either.